src/services/countries.js

```
@@ -0,0 +1,37 @@
+      headers: {
+        'X-CSCAPI-KEY': process.env.COUNTRY_API_KEY,
+      },
+    });
+
+    const sortedCities = response.data.sort((a, b) => {
+      return a.name.localeCompare(b.name);
+    });
+
+    return sortedCities;
+  } catch (error) {
+    throw new Error(`Error with getting cities in country with ${countryIso2}`);
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)